### ISSUE TITLE
Recovery from runout or pause function fails

### ISSUE THREAD

**Issue by portudork** [NARUTOfzr/Neptune_3#55]
Whenever I pause a print to change filament or trip the filament runout sensor the printer fails to resume printing (extruding filament) upon resume. The print head goes back to the last spot before pause but doesn't extrude any filament anymore while the print head moves effectively ruining any print. The pause and runout function are effectively useless due to this issue.

**Comment by NARUTOfzr** [comment]
Try making the change:
![image](https://user-images.githubusercontent.com/64059773/210299441-82c9b583-d3de-4028-814d-293e9d303f02.png)

![image](https://user-images.githubusercontent.com/64059773/210299495-6aad125b-c369-44e8-9e40-40d4887076d0.png)

Or try updating firmware V1_1.0.5-beta

**Comment by portudork** [comment]
Do you mean to uncheck that setting and remove that g code? By default it's checked and that code is already in there.

**Comment by NARUTOfzr** [comment]
Yes. This is not the configuration file uploaded by elegoo. It does not match the machine. Extrusion relative mode should not be used.




---Original---
From: ***@***.***&gt;
Date: Wed, Jan 4, 2023 12:59 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NARUTOfzr/Neptune_3] Recovery from runout or pause functionfails (Issue #55)




 
Do you mean to uncheck that setting and remove that g code? By default it's checked and that code is already in there.

**Comment by portudork** [comment]
And delete out the G92 E0 as well right? Over you confirm I'll do both and retest as soon as this print is done. Thanks for the quick response!

**Comment by NARUTOfzr** [comment]
yes. You can also comment out this line by adding ";" before it.




---Original---
From: ***@***.***&gt;
Date: Wed, Jan 4, 2023 13:19 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [NARUTOfzr/Neptune_3] Recovery from runout or pause functionfails (Issue #55)




 
And delete out the G92 E0 as well right? Over you confirm I'll do both and retest as soon as this print is done. Thanks for the quick response!

**Comment by TheAussieTinker** [comment]
I have done what you said, unchecked use relative e distance and commented out all G92 E0 lines in the gcode on PrusaSlicer. However when I open the gcode file PrusaSlicer has injected heaps of G92 E0 throughout the file that aren't commented out.

<img width="302" alt="image" src="https://github.com/NARUTOfzr/Neptune_3/assets/64671097/6ba26674-3d3a-49cc-adae-5a0d70c5c324">
